assets/src/components/cd/services/ServicesTable.tsx

```diff
@@ -56,9 +55,9 @@ export function ServicesTable({
       queryKey: 'serviceDeployments',
     },
     {
-      q: debouncedSearchString,
+      q: queryString,
```

Comment:
Why are you removing debounce here?

Reply:
It was just moved into the component

Reply:
Oh, okay. Not easy to read code here instead of IDE.